src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -125,6 +125,13 @@ MulticopterAttitudeControl::MulticopterAttitudeControl() :
 		_sensor_correction.gyro_scale_2[i] = 1.0f;
 	}
 
+	_param_handle_vtol_type = param_find("VT_TYPE");
```

Comment:
On second thought we should be more careful here. `VT_TYPE` is 0 both for tailsitters and vehicles that aren't even VTOLs.

How about only populating `_is_tailsitter` once in the first vehicle_status update that sets ORB IDs differently for VTOL vs regular MC (line ~ 262)?

Reply:
@dagar Done! See my last commit, I also cleaned the usage in the fixed wing attitude controller to be consistent.